index.html

```diff
@@ -0,0 +1,213 @@
+<!DOCTYPE html>
+<html>
+<head>
+
+    <title>Startup Matchmaker</title>
+
+<link rel="stylesheet" href="css/styles.css">
+
+</head>
+
+<body>
+
+    <header>
+
+        <h1> Startup Matchmaker </h1>
+
+    </header>
+
+    <nav>
+        <ul>
+            
+            <li class='nav-upper'>Find Developers</li>
+
+            <li class='nav-upper'>Find Designers</li>
+
+            <li class='nav-lower'>How it Works</li>
+
+            <li class='nav-lower'>Our Team</li>
+            
+            <li class='nav-lower'>Blog</li>
+
+        </ul>
+
+    </nav>
+
+ 
+    <div class='sub-head'>
+        
+        <p class='sub-head-text'> Because two brains <strike>taste</strike> <u>are</u> better than one. </p>
```

Comment:
Try to avoid using tags that style your content, style your page through css. That `taste` still needs to change its color. Styling through css gives you much more control in doing this 🔨